src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -873,6 +905,28 @@ protected DynamoModel(IStartConfiguration config)
             DynamoReady(new ReadyParams(this));
         }
 
+        private void CheckFeatureFlagTest()
+        {
+            if (DynamoModel.FeatureFlags.CheckFeatureFlag<bool>("EasterEggIcon1", false))
```

Comment:
we will need to store the possible feature flag names in some way (config file?) right ?

Reply:
why?

Reply:
I should mention that the keys, once created in launch darkly, are immutable.

Reply:
In our discussions it seemed necessary to provide a way for feature flags to work offline (Like for ALIAS)
I may have misunderstood.
It is fine for now regardless.